src/main/kotlin/no/nav/familie/ks/sak/api/BehandlingController.kt

```diff
@@ -31,4 +34,21 @@ class BehandlingController(private val behandlingService: BehandlingService) {
     fun hentBehandling(@PathVariable behandlingId: Long): ResponseEntity<Ressurs<BehandlingResponsDto>> {
         return ResponseEntity.ok(Ressurs.success(behandlingService.lagBehandlingRespons(behandlingId = behandlingId)))
     }
+
+    @PutMapping(path = ["/{behandlingId}/enhet"], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
Jeg synes det er beste bruke /enhet foran /{behandlingId}

Reply:
Tror best practice er behandling/{behandlingId}/enhet
https://restfulapi.net/resource-naming/

![image](https://user-images.githubusercontent.com/17828446/192250822-0c3cd2d3-6dc8-434e-8e0c-68807ff629cc.png)

Reply:
Kanskje til og med `behandlinger/{behandlingId}/enhet` er det beste 🤔 
![image](https://user-images.githubusercontent.com/17828446/192251228-e7ba0806-cac4-44ea-8df9-d7f5e85b52b8.png)

Reply:
hmm, stemmer. Var misforstått litt, /api/behandlinger/enhet/{behandlingId} betyr det er enhet som eier behandlingId, men det er ikke riktig. Da er /api/behandlinger/{behandlingId}/enhet er riktig. Takk for forklaringen @halvorbmundal